### PR TITLE
Fix/form

### PR DESCRIPTION
**Pull request by Falren**
add recaptcha, notification on submitting the form and change the content of mail

### CODE REVIEW

File: app/assets/javascripts/application.coffee [hunk 1]
```diff
@@ -34,21 +38,28 @@ window.submit = (form) ->
...
-  setTimeout ->
-    closeWindow()
-  , 3500
   return false
 
+alertShow = (message) -> 
+  alertMessage.innerText = message
+  alert.classList.add("showAlert");
+  setTimeout ->
+    alert.classList.remove("showAlert");
```
Comment (Falren) [inline app/assets/javascripts/application.coffee:57]: Notification on form submit

File: app/assets/stylesheets/_contact.scss [hunk 2]
```diff
@@ -161,7 +165,42 @@ $initial-margin: 5;
...
+    top: 20px;
+    border-radius: 4px;
+    border-left: 8px solid $light-violet;
+    overflow: hidden;
+    transition: all 0.2s ease-in-out 0s;
+    opacity: 0;
+    pointer-events: none;
+  }
+  .alert.showAlert{
+    opacity: 1;
```
Comment (Falren) [inline app/assets/stylesheets/_contact.scss:185]: Styles for the notification

File: app/controllers/requests_controller.rb [hunk 3]
```diff
@@ -2,11 +2,18 @@ class RequestsController < ApplicationController
   skip_before_action :verify_authenticity_token
 
   def create
-    Request.confirm(attrs).deliver_later
-    Request.notify(attrs).deliver_later
-    respond_to do |f|
-      f.html { redirect_to root_path }
-      f.js { head 200 }
+    @success = verify_recaptcha(action: 'submit', minimum_score: 0.5, secret_key: Rails.application.secrets[:recaptcha_secret_key])
+    if @success
```
Comment (Falren) [inline app/controllers/requests_controller.rb:6]: Add captcha verification and the way to handle the invalid request

File: app/views/request/confirm.slim [hunk 4]
```diff
@@ -1,3 +1,16 @@
 h1= "Hi, #{@name}"
 p= t('.message')
-span The Active Bridge Team
+span.link The Active Bridge Team
+p.link 
+  | Website: 
+  = link_to nil ,t('meta.origin_url'), target: '_blank'
+p.link 
+  | Facebook: 
+  = link_to nil, t('social.facebook'), target: '_blank'
```
Comment (Falren) [inline app/views/request/confirm.slim:9]: Email body

File: app/views/welcome/contact.html.slim [hunk 5]
```diff
@@ -27,6 +30,7 @@
       .material-input.textarea
         label for='body' =t('.message_placeholder') + ' *'
         textarea#body name='contact[message]' placeholder=' ' minlength='10' maxlength='500' required='true'
+        = recaptcha_v3(action: 'submit', site_key: Rails.application.secrets[:recaptcha_site_key])
```
Comment (Falren) [inline app/views/welcome/contact.html.slim:33]: Client side verification

**Review by galulex — APPROVED** [review]
+1

**Pull request merged**